app/models/DriverManager.py

```diff
@@ -0,0 +1,33 @@
+from uuid import uuid4
+
+from rtree import index
+from geopy.distance import geodesic
+from typing import Tuple, Dict
+
+from app.models.Driver import Driver
+
+
+class DriverManager:
+    def __init__(self):
+        self.drivers: Dict[str, Driver] = {}
+        self.rtree_index = index.Index()
+        self.id_counter = 0
+
+    def addDriver(self, driver: Driver):
+        self.drivers[driver.driverId] = driver
+        self.id_counter += 1
+        self.rtree_index.insert(self.id_counter, (*driver.driverLocation, *driver.driverLocation), obj=driver.driverId)
```

Comment:
whats happening in this section ? 

`self.rtree_index.insert(self.id_counter, (*driver.driverLocation, *driver.driverLocation), obj=driver.driverId)`

Why are we passing in the same object twice ? And what does * do before the `driver` ?